### DISCUSSION TITLE
Dbeaver

### DISCUSSION THREAD

**Discussion by dbeaver12** [dbeaver/dbeaver#19684] [General]
### Description

Hi..

As part of new software research in Manchester Metropolitan university, we would like to check on these prior to use in one of our departments0

1. when was software last updated
2. Is software still on active development
3. Are the software provider committed to patching the software in event of security incident or known security vulnerabilities.

Please could answer the above mentioned question in regards to the dbeaver.

Thanks and Regards,

Jesu

### DBeaver Version

nil

### Operating System

nil

### Database and driver

nil

### Steps to reproduce

nil

### Additional context

nil

**Comment by Destrolaric** [comment]
@dbeaver12 
1. Latest release was three days ago, DBeaver community edition is updated every two weeks.
2. The software is stable and is actively maintained by the team
3. Yes, in the event of a security incident or discovered vulnerability, the software will be updated, in some cases even rereleased out of the standard release cycle.
